### DISCUSSION TITLE
How can i change column "current_date"  every minute to the current date ?

### DISCUSSION THREAD

**Discussion by HaitarDev** [supabase/supabase#18383] [Questions]
I try to use pg_cron and it work with me when i want to insert a row every minute but when i want to update the current_date column it didnt work . 
`-- Create a scheduled job to update the current_date column every minute
SELECT cron.schedule('*/1 * * * *', $$
  UPDATE membership
  SET current_date = NOW();
$$);
`

**Comment by GaryAustin1** [comment]
You should check the Postgres logs in the dashboard.  Normally Supabase requires a WHERE clause on update and delete to stop accidentally impacting an entire table.
https://supabase.com/docs/guides/database/extensions/pg-safeupdate

**Reply by HaitarDev** [reply]
i just logs this function: SELECT cron.schedule( 'update-column-every-minute', '* * * * *', -- every minute $$ UPDATE membership SET current_date = NOW() WHERE 1=1; -- Modify 'some_condition' to match your criteria $$ );
and give me this error : {
"code": 400,
"errors": [
{
"domain": "global",
"location": "q",
"locationType": "parameter",
"message": "Syntax error: Illegal input character "$" at [1:9156]",
"reason": "invalidQuery"
}
],
"message": "Syntax error: Illegal input character "$" at [1:9156]",
"status": "INVALID_ARGUMENT"
}

**Reply by GaryAustin1** [reply]
It is very hard to tell about your code as you are not formatting it.  Surround it with triple back ticks on their own lines.  If you put SQL after the first three ticks then it will highlight sql.  

```SQL
SELECT cron.schedule( 'update-column-every-minute', '* * * * *', -- every minute 
$$ UPDATE cat SET created_at = NOW() WHERE 1=1; -- Modify 'some_condition' to match your criteria 
$$ );
```
This worked for me where I just changed your table name and column to ones I had.
